lib/matplotlib/text.py

```diff
@@ -1657,17 +1657,41 @@ def set_figure(self, fig):
 
 
 class OffsetFrom(object):
+    '''Callable helper class for working with `Annotation`
```

Comment:
I'm normally a proponent of single quotes because they're easier to type, but isn't the convention to use double quotes for docstrings? Not that that's a huge difference or anything...